internal/cmd/get/get.go

```diff
@@ -43,6 +44,17 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
+func validateName(name string) (bool, string) {
+	const regexpString = `^[a-z0-9_-]+$`
+
+	matched, err := regexp.MatchString(regexpString, name)
+	if err != nil {
+		log.Fatalf("matching string '%s', using regexp '%s' failed with error: %s", name, regexpString, err)
```

Comment:
not sure if it's the best practice to log/return regex explicitly (looks more like a debug thing to me), but I guess it's servers the same purpose as saying "the name %s should consist of only alphanumeric characters, underscore and hyphen"